cfps/tasks.py

```
@@ -0,0 +1,80 @@
+            subject=subject,
+            message=message,
+            from_email=settings.DEFAULT_FROM_EMAIL,
+            recipient_list=[submission.submitter.email],
+            fail_silently=False,
+        )
+        logger.info(
+            "CFP status email sent to %s (status=%s)",
+            submission.submitter.email,
+            submission.status,
+        )
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)